src/Uno.UI/UI/Xaml/Controls/ItemsControl/ItemsControl.cs

```diff
@@ -92,7 +92,7 @@ void Initialize()
 
 			_items.VectorChanged += (s, e) =>
 			{
-				OnItemsChanged(null);
+				OnItemsChanged(e);
```

Comment:
We do have unit tests for that class, IIRC, but it does not test the arguments.

https://github.com/unoplatform/uno/blob/f88bff0853a4e2d8467273ce6d4005fc04182746/src/Uno.UI.Tests/Windows_UI_XAML_Controls/ItemsControlTests/Given_ItemsControl.cs#L62

Reply:
added a check for the args